x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.test.tsx

```diff
@@ -59,11 +60,14 @@ describe('PrecisionSlider', () => {
 
   describe('Step Description', () => {
     it('is visible when there is a step description', () => {
-      setMockValues({ ...MOCK_VALUES, precision: 10 });
```

Comment:
setting `precision` instead of `searchSettings.precision` meant that this `setMockValues` wasn't actually doing anything and `searchSettings.precision` remained at the value `2`

Reply:
Nice catch!